runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/config/ConfigWriter.java

```diff
@@ -49,20 +57,41 @@ public void write(
         NamespaceConfig namespace,
         Writer writer)
     {
-        write0(namespace, writer);
+        write0(namespace, writer, NOOP_PATCH, List.of());
+    }
+
+    public void write(
+        NamespaceConfig namespace,
+        Writer writer,
+        JsonPatch patch,
+        List<String> unquotables)
+    {
+        write0(namespace, writer, patch, unquotables);
     }
 
     public String write(
         NamespaceConfig namespace)
     {
         StringWriter writer = new StringWriter();
-        write0(namespace, writer);
+        write0(namespace, writer, NOOP_PATCH, List.of());
+        return writer.toString();
+    }
+
+    public String write(
+        NamespaceConfig namespace,
+        JsonPatch patch,
+        List<String> unquotedEnvVars)
+    {
+        StringWriter writer = new StringWriter();
+        write0(namespace, writer, patch, unquotedEnvVars);
         return writer.toString();
     }
 
     private void write0(
         NamespaceConfig namespace,
-        Writer writer)
+        Writer writer,
+        JsonPatch patch,
+        List<String> unquotedEnvVars)
```

Comment:
Let's not leak the concept of unquoted env vars into the config writer.
Instead please post-process the resulting YAML string in the config command.

Reply:
Created the method `unquoteEnvVars` in `OpenApiHttpProxyConfigGenerator`, I hope it's OK.